it/controllers/ForecastTaxCalcSummaryControllerISpec.scala

```diff
@@ -1,18 +1,75 @@
 
 package controllers
 
+import audit.models.ForecastTaxCalculationAuditModel
+import auth.MtdItUserWithNino
 import config.featureswitch.ForecastCalculation
 import helpers.ComponentSpecBase
-import helpers.servicemocks.IncomeTaxCalculationStub
-import play.api.http.Status.{INTERNAL_SERVER_ERROR, NOT_FOUND, OK}
-import testConstants.BaseIntegrationTestConstants.{testNino, testYear}
-import testConstants.NewCalcBreakdownItTestConstants.{liabilityCalculationModelError, liabilityCalculationModelSuccessful}
+import helpers.servicemocks.{AuditStub, IncomeTaxCalculationStub}
+import models.liabilitycalculation.{EndOfYearEstimate, IncomeSource}
+import play.api.http.Status.{NOT_FOUND, OK}
+import play.api.test.FakeRequest
+import testConstants.BaseIntegrationTestConstants.{testMtditid, testNino, testYear}
+import testConstants.NewCalcBreakdownItTestConstants.liabilityCalculationModelSuccessful
+import uk.gov.hmrc.auth.core.AffinityGroup.Individual
+
+object ForecastTaxSummaryControllerTestConstants {
+  val mtdItUser: MtdItUserWithNino[_] = MtdItUserWithNino(testMtditid, testNino, None, None, Some("1234567890"),
+    Some("12345-credId"), Some(Individual), None)(FakeRequest())
+
+  val taxableIncome = 12500
```

Comment:
I would assume this should be an amount which should be BigDecimal ?

Reply:
This is an Int in the IncomeSource case class so I think this is okay?